docs/source/examples/shapes_images_positions.rst

```diff
@@ -16,20 +16,58 @@ Shapes, Images and Positions
             self.add(d, c, s, t)
             self.wait(1)
 
-.. manim:: Example1ImageFromArray
+
+.. manim:: ShapesExample
+    
+    class ShapesExample(Scene):
+        
+        def construct(self):
+            
+            circle = Circle(radius= 1, color=BLUE)
+            
+            dot = Dot()
+            dot2= dot.copy().shift(RIGHT)
+            self.add(dot)
+            
+            line=Line(np.array([3,0,0]),np.array([5,0,0]))
+            self.add(line)
+            
+            
+            self.play(GrowFromCenter(circle))
+            self.play(Transform(dot,dot2))
+            self.play(MoveAlongPath(dot,circle), run_time= 2, rate_func=linear)
+            self.play(Rotating(dot, about_point=np.array((2, 0, 0.))), run_time=1.5)
+            self.wait()
+
+
+.. manim:: ImageFromArrayExample
     :save_last_frame:
 
-    class Example1ImageFromArray(Scene):
+    class ImageFromArrayExample(Scene):
         def construct(self):
             image = ImageMobject(np.uint8([[0, 100, 30, 200],
                                            [255, 0, 5, 33]]))
             image.set_height(7)
             self.add(image)
```

Comment:
This example is very redundant to GradientImageFromArrayExample, so I'd say we move it  in the reference

Reply:
What does «moving to reference» mean ? Like writing it in the docstrings ?

Reply:
Like this here:
https://github.com/ManimCommunity/manim/blob/1b12f076be49a677add97346eed6900e3cf18873/manim/mobject/value_tracker.py#L23
that will render the example for the reference here:
https://manimce.readthedocs.io/en/latest/reference/manim.mobject.value_tracker.ValueTracker.html#manim.mobject.value_tracker.ValueTracker
It will then appear here.
And you can put the first image object  here:
https://github.com/ManimCommunity/manim/blob/1b12f076be49a677add97346eed6900e3cf18873/manim/mobject/types/image_mobject.py#L51